### PR TITLE
feat: anchor ink ball style

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Change Anchor ink ball style to line.       |
| 🇨🇳 Chinese |  Anchor 组件 ink ball 改为线型样式。         |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/affix/index.en-US.md [hunk 1]
```diff
@@ -26,12 +26,12 @@ Please note that Affix should not cover other content on the page, especially wh
...
-| offsetBottom | Offset from the bottom of the viewport (in pixels) | number | - |
-| offsetTop | Offset from the top of the viewport (in pixels) | number | 0 |
-| target | Specifies the scrollable area DOM node | () => HTMLElement | () => window |
-| onChange | Callback for when Affix state is changed | (affixed?: boolean) => void | - |
+| Property     | Description                                        | Type                        | Default      |
+| ------------ | -------------------------------------------------- | --------------------------- | ------------ |
+| offsetBottom | Offset from the bottom of the viewport (in pixels) | number                      | -            |
+| offsetTop    | Offset from the top of the viewport (in pixels)    | number                      | 0            |
+| target       | Specifies the scrollable area DOM node             | () => HTMLElement           | () => window |
+| onChange     | Callback for when Affix state is changed           | (affixed?: boolean) => void | -            |
```
Comment (afc163) [inline components/affix/index.en-US.md:34]: https://github.com/prettier/prettier/issues/5651 这个问题怎么又出现了。

Comment (afc163) [reply components/affix/index.en-US.md:34]: @zombieJ 好像不应该改这个 https://github.com/ant-design/ant-design/blame/9c670dfdf65dfbad217827408b8b90630a34536c/.prettierrc#L13-L18

https://github.com/prettier/prettier/commit/a9d2c4146494e429f174111b989f21ef5e28bb36#diff-8940c253e1f0b256506df2285357614ae8d078056d822235e63a20058b400e55R45

File: components/anchor/index.en-US.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ---
 category: Components
 title: Anchor
-cover: https://gw.alipayobjects.com/zos/alicdn/_1-C1JwsC/Anchor.svg
+cover: https://gw.alipayobjects.com/zos/alicdn/_1-C1JwsC/Anchor.svg7
```
Comment (li-jia-nan) [inline components/anchor/index.en-US.md:4]: 这个链接好像不对

File: components/anchor/index.en-US.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 category: Components
 title: Anchor
-cover: https://gw.alipayobjects.com/zos/alicdn/_1-C1JwsC/Anchor.svg7
+cover: https://gw.alipayobjects.com/zos/alicdn/_1-C1JwsC/Anchor.svg
```
Comment (afc163) [inline components/anchor/index.en-US.md:4]: @arvinxx 求个新图。。

**Comment by afc163** [comment]
右边的貌似歪了

图片

**Comment by afc163** [comment]
图片

高亮节点没有跟随滚动更新。

图片

**Review by zombieJ — APPROVED** [review]

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**